consensus/istanbul/backend.go

```diff
@@ -41,8 +43,8 @@ type SignerFn func(accounts.Account, string, []byte) ([]byte, error)
 // backing account using BLS with a direct or composite hasher
 type BLSSignerFn func(accounts.Account, []byte, []byte, bool) (blscrypto.SerializedSignature, error)
 
-// Backend provides application specific functions for Istanbul core
-type Backend interface {
+// BackendForCore provides the Istanbul backend application specific functions for Istanbul core
```

Comment:
A more ideomatic way to do this would be to move this interface into the `consensus/istanbul/core` package so that it lives next the code that needs it, and the name `Backend` will be unambigous.

Reply:
Done.